src/cloud-cli/meltano/cloud/cli/history.py

```diff
@@ -0,0 +1,139 @@
+"""Meltano project execution history."""
+
+from __future__ import annotations
+
+import asyncio
+import datetime
+import json
+import typing as t
+
+import click
+import tabulate
+
+from meltano.cloud.api.client import MeltanoCloudClient
+from meltano.cloud.cli.base import cloud
+
+if t.TYPE_CHECKING:
+    from meltano.cloud.api.config import MeltanoCloudConfig
+    from meltano.cloud.api.types import CloudExecution
+
+PAGE_SIZE = 100
```

Comment:
What's the justification behind this page size? Does it correspond to anything configured in the backend?